scvi_in_R.ipynb

```diff
@@ -0,0 +1,731 @@
+{
```

Comment:
<p>n_epochs is automatic now</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/YosefLab/scvi-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Yea, I wanted to show how to actually set it though since you need to do this as.integer thing. </p>